README.md

```diff
@@ -77,9 +77,20 @@ To deploy SMP server on [Linode](https://www.linode.com/):
 
 Please submit an [issue](https://github.com/simplex-chat/simplexmq/issues) if any problems occur.
 
-## 🚧 Deploy SMP server on DigitalOcean 🚧
+[<img alt="digitalocean" src="./img/digitalocean.svg" align="right" width="200">](https://cloud.digitalocean.com)
 
-Coming soon.
+## Deploy SMP server on DigitalOcean
+
+To deploy SMP server on [DigitalOcean](https://cloud.digitalocean.com):
+- Create a DigitalOcean account or login with an already existing one;
+- Click [create Droplet](https://cloud.digitalocean.com/droplets/new?size=s-1vcpu-1gb&region=fra1);
+- Choose the region and plan according to your requirements (for regular use cheapest Regular option should be sufficient).
+- Provide ssh key to be able to connect to your Droplet via ssh. This step is required, because you will need to login to your Droplet and get a SMP server public key hash (`hash` later) either from the welcome message or from the file `/etc/opt/simplex/simplex.conf` on your Droplet after SMP server starts.
+- Create your Droplet.
```

Comment:
`Create your Droplet` -> `Confirm Droplet creation`? I'd move it to the previous step - you're focusing too much on following sites' UIs letter to letter - if DigitalOcean slightly adjust their forms (for example by adding one more choice) this tutorial is out of sync. E.g.

```Provide ssh key and confirm Droplet creation. Ssh key is required for you to login to the Droplet and retrieve and SMP server public key hash either from the welcome message or `/etc/opt/simplex/simplex.conf` after server start```